packages/dapp/src/hooks/useEncryptedData.ts

```diff
@@ -0,0 +1,22 @@
+import { useState, useCallback } from 'react';
+
+export type UseEncryptedDataHook = [
+  encryptionAccount: string,
+  setEncryptedData: Function
+];
+
+let initialEncryptedData = localStorage.getItem("encryptedData") as string;
+if (!initialEncryptedData) {
+  initialEncryptedData = "";
+}
+
+export const useEncryptedData = (): UseEncryptedDataHook => {
+  const [encryptionAccount, setEncryptedData] = useState(initialEncryptedData);
+
+  const switchEncryptedData = useCallback((newEncryptedData) => {
```

Comment:
does it make sense to use `useCallback` hook while a whole function seems quite static?